src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+
+  if (days >= 7) {
+    price -= 50;
+  } else if (days >= 3) {
+    price -= 20;
```

Comment:
Replace the literal '3' with a constant to give it a meaningful name and replace '20' with a constant for the short term discount. For example, `const SHORT_TERM_DAYS = 3;` and `const SHORT_TERM_DISCOUNT = 20;`.